client/layout/community-translator/launcher.jsx

```diff
@@ -29,7 +29,7 @@ class TranslatorLauncher extends React.PureComponent {
 		infoDialogVisible: false,
 		firstActivation: true,
 		isActive: translator.isActivated(),
-		isEnabled: translator.isEnabled(),
+		isEnabled: true,
```

Comment:
Is the `translator.isEnabled()` call an old check we don't need any more?

Reply:
Ahh, this was meant to be chopped out. I just hardcoded it to make sure it'd show up each time :)
I'll remove it and put the code change in the test plan 👌